intl/messages/en.json

```diff
@@ -64,10 +64,15 @@
       }
     ]
   },
-
   "gettingStarted": {
     "sectionTitle": "Getting Started",
-    "sectionDesc": "TODO(Hugo+Marco)",
+    "sectionDesc": "IPFS allows you to store and distribute data, addressing it by its hash. If the data changes, the hash changes as well.",
+    "addDataToIPFS": "Adding data to IPFS",
+    "output": "Output",
+    "getDataFromIPFS": "Getting data from IPFS",
+    "usingJavascript": "Using Javascript in the Browser or Node.js",
+    "usingCli": "Using the CLI",
+    "usingGateway": "Using the HTTP Gateway",
     "list": [
```

Comment:
If Getting Started steps are no more editable, remove this `list` array. If they're still editable, add each step to the array, please.